README.md

```diff
@@ -1 +1,47 @@
-# open-ce-python
\ No newline at end of file
+[![Open-CE-Python Stars](https://img.shields.io/github/stars/open-ce-python?style=social)](https://github.com/open-ce/open-ce-python/stargazers)
+
+<p align="center">
+  <img src="https://avatars0.githubusercontent.com/u/68873540?s=400&u=a02dc4156e50cdffb23172aba7133e44381885d4&v=4" alt="Open-CE Logo" width="30%">
+</p>
+
+[![Python Support](https://img.shields.io/badge/python-3.10-blue.svg)](#requirements)
+[![Architecture Support](https://img.shields.io/badge/architecture-ppc64le%20%7C%20)](#)
+[![GitHub Licence](https://img.shields.io/github/license/open-ce/open-ce.svg)](LICENSE)
+---
+
+This is a sub-project of the Open-CE project for building python optimized for Power10 architecture.
+
+This repository contains an Open-CE environment file that builds optimized python using the [python feedstock](https:///github.com/open-ce/python-feedstock). Open-CE-Python currently supports the following:
+
+| | Supported Versions |
+| --- | --- |
+| Architecture | Power |
+| Python | 3.10 |
+
+## GETTING STARTED
+
+### Requirements
+
+* `conda`
+  * The conda tool can either be installed through [Anaconda](https://www.anaconda.com/products/individual#Downloads) or [Miniconda](https://docs.conda.io/en/latest/miniconda.html).
+
+* `Redhat GCC 11`
+  Redhat's `gcc-toolset-11` package is needed which can be installed via `yum install -y gcc-toolset-11`. To install this, we need to have RHEL 8.x with Power9/Power10 architecture.
```

Comment:
~To install this, we need to have RHEL 8.x with Power9/Power10 architecture.~  REHL 8.x with Power 9 or Power 10 architecture is required.

Reply:
Done. Thanks.